import old samples and add new README (#524)
GoogleChrome/chrome-extensions-samples: README.md

# Chrome Extensions samples

> Official samples for Chrome Extensions (and the [deprecated](https://developer.chrome.com/apps/migration) Chrome Apps platform).

Reviewer: I think instead of the parenthetical it might be good to say "Official samples for Chrome Extensions and the Chrome Apps platform" and below that add a note that Chrome apps are deprecated and link to the blog post to 'learn more'.

Author: Great.

For more information, see [Chrome Developers](https://developer.chrome.com).

## Samples

The directory structure is as follows:

> * api/ - trivial extensions focused on a single API package

Reviewer: ```suggestion
* api/ - extensions focused on a single API package
```

just thinking about this from an inclusive language perspective. I think words like 'trivial' or 'simple' may be unwelcoming to beginners (I know that's not the intent of the use of trivial here, but I think it can be read that way).

Author: This is imported from before. Happy to make the change.

> * howto/ - simple extensions showing how to perform a particular task

Reviewer: ```suggestion
* howto/ - extensions that show how to perform a particular task
```

* tutorials/ - multi-step walkthroughs referenced inline in the docs
* extensions/ - full featured extensions spanning multiple API packages
* apps/ - deprecated Chrome Apps platform (not listed below)

> To experiment with these samples, please download this repo and use 'Load Unpacked Extension'.

Reviewer: ```suggestion
To experiment with these samples, please clone this repo and use 'Load Unpacked Extension'.
```
